tests/scripts/run_models.sh

```diff
@@ -15,6 +15,8 @@ fi
 if [[ $ARCH_NAME == "wormhole" ]]; then
   env pytest tests/ttnn/integration_tests/stable_diffusion
   env pytest tests/ttnn/integration_tests/unet
+  env pytest models/experimental/mamba/tests
```

Comment:
Hey @kpaigwar 

So I would actually recommend putting in a different if block with the proper arch name. There is actually a typo here - it should be `wormhole_b0` and not `wormhole`. I believe the original authors of this if block did not check this.

I would also recommend putting in a `@skip_for_grayskull` decorator with a reason in English to be sure that people see what's going on when they skip it on GS.

```
if [[ $ARCH_NAME == "wormhole_b0" ]]; then
  env pytest models/experimental/mamba/tests
fi
```

here. And then you should run the fast dispatch nightly tests: https://github.com/tenstorrent-metal/tt-metal/actions/workflows/full-regressions-and-models.yaml

Verify that it is running on CI and paste the passing link in this PR.

It should be one of the first models run, so you should have pretty quick feedback.

Reply:
We are currently working on skipping any failing test in that pipeline, but you should not be blocked from doing your work and your mamba test should be one of the first run on `wormhole_b0, models` variant of fast dispatch pipeline.

Reply:
Hey @tt-rkim, thanks for the review. I added suggested changes. And here is the link for fast-dispatch nightly run https://github.com/tenstorrent-metal/tt-metal/actions/runs/8461855598